src/translations/de/translations.json

```diff
@@ -240,6 +240,22 @@
       "body": "Du erhältst eine Zahlungsbestätigung an die E-Mail-Adresse {{useremail}}.",
       "title": "Warte auf die Zahlungsbestätigung"
     },
+    "25": {
+      "body": "Non è stato addebitato alcun importo.\n Se il problema persiste, prova a usare un altro metodo o gestore del pagamento.",
+      "title": "Il pagamento non è andato a buon fine"
+    },
+    "116": {
+      "body": "Non è stato addebitato alcun importo. Prima di riprovare, aggiungi fondi alla o usa un altro metodo di pagamento.",
+      "title": "Il saldo è insufficiente"
+    },
+    "117": {
+      "body": "Non è stato addebitato alcun importo. Il codice (CVV o CVC) ha 3 cifre ed è riportato sul retro della carta. Se usi American Express, il codice (CID) ha 4 cifre ed è sul fronte.",
+      "title": "Il codice di sicurezza inserito è errato"
+    },
+    "121": {
+      "body": "Non è stato addebitato alcun importo. Prima di riprovare, modifica il limite di spesa o usa un altro metodo di pagamento.",
+      "title": "Hai superato il limite di spesa"
+    },
```

Comment:
why adding those it translation for all languages? If i remember correctly italian is the default fallback language so we can avoid adding those translation here and wait for translations for all languages in a separate pr. what do you think?

Reply:
We can do that, but we have to remove the unit tests for all translations except Italian, because they fail if some translations are missing. 
If we do that they will be to be restored in the next PR.

Reply:
waiting for translations ? so the PR is on hold

Reply:
Right now it was decided to use only Italian as language. [Jira task](https://pagopa.atlassian.net/browse/CHK-3583?focusedCommentId=188993)